tensorflow/python/keras/optimizer_v2/nadam.py

```diff
@@ -59,6 +59,18 @@ class Nadam(optimizer_v2.OptimizerV2):
 
   References
     See [Dozat, T., 2015](http://cs229.stanford.edu/proj2015/054_report.pdf).
+    
+  Usage Example:
```

Comment:
I would really like to see examples like this:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/adam.py#L73-L79